sdk/agrifood/agrifood-farming-rest/review/agrifood-farming.api.md

```diff
@@ -1175,50 +1341,53 @@ export type FarmBeatsRestClient = Client & {
 
 // @public (undocumented)
 export interface Farmer {
-    createdDateTime?: Date;
+    createdDateTime?: Date | string;
     description?: string;
     eTag?: string;
     id?: string;
-    modifiedDateTime?: Date;
+    modifiedDateTime?: Date | string;
     name?: string;
-    properties?: FarmerPropertiesDictionary;
```

Comment:
I don't see any other instances of this type name other than its deletion here. Did we mean to change these from named property bags to anonymous maps to `any`?